src/Files.App/App.xaml

```diff
@@ -30,8 +30,9 @@
 				<XamlControlsResources xmlns="using:Microsoft.UI.Xaml.Controls" />
 
 				<!--  Styles for the custom icons  -->
-				<ResourceDictionary Source="/ResourceDictionaries/PathIcons.xaml" />
-				<ResourceDictionary Source="/UserControls/SideBar/SideBarControls.xaml" />
+				<ResourceDictionary Source="ms-appx:///ResourceDictionaries/AppThemeGeneric.xaml" />
```

Comment:
Unless this is used in more than three or four xaml files, we shouldn't include it in app.xaml.

Reply:
I had to reference App.Resources from StatusCenterItem. Do you know another way to access to it without adding it in App class?

Reply:
We can move resources defined above in App.xaml to AppGeneric.xaml though.

Reply:
> I had to reference App.Resources from StatusCenterItem. Do you know another way to access to it without adding it in App class?

Not that I know. It'll be a good idea to add a comment so I don't forget and make some breaking changes.

Reply:
Sure!